aptos-move/block-executor/src/task.rs

```diff
@@ -94,13 +94,40 @@ pub trait TransactionOutput: Send + Sync + Debug {
     /// Get the events of a transaction from its output.
     fn get_events(&self) -> Vec<<Self::Txn as Transaction>::Event>;
 
+    /// Get resource group updates.
```

Comment:
Is this a get function? The function doesn't have any return values.

Reply:
good catch!